### DISCUSSION TITLE
How to know go version

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#688] [General]
Oleg, with your new releases it is possible to see version number.  Perfect. 
quick question: could you so gently tell me how to identify the release on old go installation ?
Maybe some read e/config/release file ?… thanks in advance.

**Comment by oleg68** [comment]
- In Linux, you can do ``rpm -qa grandorgue`` or ``dpkg -s grandorgue`` depending on the linux distribution
- You can run GrandOrgue. The release (first 4 numbers) will appeare in the title

Unfortunally, there was no any file containing the full version (inctluding the build version)
